.gitbook/5-oracles/1-operate-oracle-nodes/4-oracle-registration.md

```diff
@@ -0,0 +1,96 @@
+# Oracle Registration
+
+## Get Trusted Block Information
+
+To request oracle registration to Panacea, trusted block information(height and hash), which will be used for light client, is required and need to be verified by other registered oracle.
```

Comment:
It would be good to write the reason for using the light client later in `0-about-dep-oracle`, and connect it with a link.

Reply:
Right, that's what I'm going to do in the future. I'll keep this in mind.